week-4-tests-and-persistence/w04e03-nest-pagination/src/shared/pagination.decorator.ts

```diff
@@ -0,0 +1,19 @@
+import { createParamDecorator, ExecutionContext } from '@nestjs/common';
+import { Request } from 'express';
+import { Pagination } from './Pagination';
+
+export const GetPagination = createParamDecorator(
+  (data, ctx: ExecutionContext): Pagination => {
+    const req: Request = ctx.switchToHttp().getRequest();
+
+    const paginationParams: Pagination = {
+      skip: 0,
+      limit: 10,
```

Comment:
to `10` to _Magic number_ - fajnie jakbyś z tego zrobił stałą gdzieś wyżej:

```typescript
const DEFAULT_LIMIT = 10;
```

Reply:
Faktycznie nie powinno być takich `magic numbers` w kodzie. Poprawione 👍